packages/docusaurus-theme-classic/src/theme/SkipToContent/index.tsx

```diff
@@ -14,7 +14,7 @@ import styles from './styles.module.css';
 export default function SkipToContent(): JSX.Element {
   const {containerRef, handleSkip} = useSkipToContent();
   return (
-    <div ref={containerRef} role="region">
+    <div ref={containerRef} role="region" aria-label="Skip to main content">
```

Comment:
Could you add a translation? It would share the same key as the main content.

Reply:
Should be fixed